src/types.tsx

```diff
@@ -371,24 +371,47 @@ export interface SearchBarProps {
    * The auto-capitalization behavior
    */
   autoCapitalize?: 'none' | 'words' | 'sentences' | 'characters';
+  /**
+   * Automatically focuses search bar on mount
+   *
+   * @platform android
+   */
+  autoFocus?: boolean;
   /**
    * The search field background color
    */
   barTintColor?: string;
   /**
    * The text to be used instead of default `Cancel` button text
+   *
+   * @platform ios
    */
   cancelButtonText?: string;
-
+  /**
+   * Specifies whether the back button should close text input or not.
```

Comment:
```suggestion
   * Specifies whether the back button should close search bar's text input or not.
```

Would this be better?